web/src/lib/components/photos-page/asset-grid.svelte

```diff
@@ -34,6 +35,7 @@
   export let isShared = false;
   export let album: AlbumResponseDto | null = null;
   export let isShowDeleteConfirmation = false;
+  export let isAllFavorite: boolean | null = null;
```

Comment:
Can you explain the use case of the variable a bit?

Reply:
Of course!

The idea is to have only one reactive declaration to know if the selected assets need to be favored or unfavoured. 

But currently, not all assets can be favored, so if `isAllFavorite` is null, we don't allow that shortcut on that asset-grid view.